src/lib/strings/url-helpers.ts

```diff
@@ -145,6 +146,13 @@ export function isBskyListUrl(url: string): boolean {
   return false
 }
 
+export function isBskyDownloadUrl(url: string): boolean {
+  if (isExternalUrl(url)) {
+    return false
+  }
+  return url.endsWith('/download')
+}
```

Comment:
Not sure that endsWith is super safe here since someone like us might toss some query params on there. What do you think about `url.startsWith(BSKY_DOWNLOAD_URL)`?